lua/gitlab/state.lua

```diff
@@ -232,6 +399,54 @@ M.merge_settings = function(args)
     )
   end
 
+  M._get_nested_field = function(table, field)
```

Comment:
Let's pull this into the utils file, it could be generally useful!

Reply:
Done.